jpa/eclipselink.jpa.test/src/it/java/org/eclipse/persistence/testing/tests/jpa21/advanced/StoredProcedureQueryTestSuite.java

```diff
@@ -1063,6 +1064,33 @@ public void testStoredProcedureQuerySysCursor2() {
         }
     }
 
+    /**
+     * Tests a StoredProcedureQuery using a system cursor. Also tests
+     * getParameters call AFTER query execution. Parameters are passed via position.
+     */
+    public void testStoredProcedureQuerySysCursor2_Positional() {
```

Comment:
This test doesn't test "getParameters", it tests "getResultList()" as opposed to the other tests (testStoredProcedureQuerySysCursor_Named/testStoredProcedureQuerySysCursor_Positional) that test getOutputParameterValue(String name)/getOutputParameterValue(int index)

testStoredProcedureQuerySysCursor2_Named above this isnt testing getResultList(), it's testing getOutputParameterValue() as well. Maybe we need to make that distinction in the test names so we can cover everything.